src/components/KSelect/KSelect.cy.ts

```diff
@@ -387,6 +411,7 @@ describe('KSelect', () => {
     const placeholderText = 'Placeholder text'
     const itemLabel = 'Label 1'
     const itemValue = 'label1'
+    const itemSlotContent = 'I am overwritten by selected-item slot'
```

Comment:
We have a test case which ensures `selected-item-template` works as expected in `autosuggest` mode and that should be enough since `reuseItemTemplate ` is essentially the same.